Userland/cal.cpp

```diff
@@ -21,29 +36,139 @@ int get_number_of_days(int month, int year)
     return is_long_month ? 31 : 30;
 }
 
+void append_to_print(char* buffer, int row, int column, char* text)
+{
+    int starting_point = (line_width * row) + (column * column_width);
+    for (int i = 0; text[i] != '\0'; i++) {
+        buffer[starting_point + i] = text[i];
```

Comment:
This is okay, but it would be more idiomatic to use pointers:
```cpp
for (char* dest = &buffer[starting_point]; *text; text++, dest++)
    *dest = *text;
```